lib/template-processing.js

```diff
@@ -77,15 +77,27 @@ const processPackageJson = (name, targetPath, { xdn, ...config } = {}) => {
     delete packageJson.scripts['rsf:link']
   }
   // remove react-storefront team from the deploy script, so it defaults to the user's own team:
-  packageJson.scripts.deploy = packageJson.scripts.deploy.replace('xdn deploy react-storefront', 'xdn deploy')
+  packageJson.scripts.deploy = packageJson.scripts.deploy.replace(
+    'xdn deploy react-storefront --environment=production',
```

Comment:
I had to remove the environment in the package.json to get the deployment script working correctly. Is this okay, or do you think the yaml should be changed to use `xdn deploy` instead of `npm run deploy`?

Reply:
Yep, what you did was right, I've been making the same change in my demos.